### ISSUE TITLE
Incorrect "not for resale" notice?

### ISSUE THREAD

**Issue by Technoturnovers** [TeamShinkansen/Hakchi2-CE#459]
On a modded system, a default install includes a legal notices menu which claims in all-capitals "THIS MODDED SYSTEM IS NOT FOR RESALE!", and a warning against 'scams' not unlike the old HackMii stuff like the Homebrew Channel on Wii and whatnot. However, uhhh, unlike that, this software isn't proprietary, it uses GPL and or permissively licensed components with no non-commercial use restrictions in any of the dependencies as far as I'm aware. So legally speaking, users have the right to resell this software provided that source is given upon request (or just period, in the case of the permissive components). This actually bears out with the aforementioned HackMii stuff- when it got re-released as open source a while back (minus some extraneous components), the "not for resale" notice got removed in accordance with said open source license.

tl;dr: This warning is legally toothless, and might even be legally problematic (although I don't know much about that). Adding a "not for resale" notice on top of 3rd-party dependencies such as 7zip feels especially presumptuous to me, since that misleads about the license terms of _other developers_ software, not just Hakchi itself.
